src/mock-data/author-post.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "601798f28301cc0600f09303",
+  "style": "article:v2:default",
+  "slug": "my-2020-lighthouse-keeper",
+  "leading_image_portrait": {},
+  "hero_image": {
+    "filetype": "image/jpeg",
+    "resized_targets": {
+      "mobile": {
+        "height": 533,
+        "width": 800,
+        "url": "https://storage.googleapis.com/twreporter-multimedia/images/20210203160708-bd3cab136eb567f4a77d25267639e37c-mobile.jpg"
```

Comment:
use `https://www.twreporter.org/images/` as the route prefix.

Reply:
Updated.